app/controllers/attributes_controller.rb

```diff
@@ -41,9 +45,7 @@ def get_attributes_from_params(attributes, permission_level:, is_hash: false)
     local_attributes = attributes.select { |name| user_attributes.stored_locally? name }
     remote_attributes = attributes.reject { |name| user_attributes.stored_locally? name }
 
-    raise NotImplementedError if local_attributes.any?
-
-    remote_attributes
+    [local_attributes, remote_attributes]
```

Comment:
how are remote attributes sorted?
Are there any expecrations we should maintain here?

LIkes sort these alphabetically at this point to intersperse the local and remote attributes?

Reply:
What would be the benefit of that?